scripts/perf-run-all-scenarios

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+for SCENARIO in "encoder" "span" "tracer" "django_simple" "flask_simple" "threading"; do
```

Comment:
missing `startup`? is there a better way we can get this list? maybe all directories in the benchmarks directory minus the tooling ones?

Reply:
We should update startup with the new framework and then add here. Not sure if it'll work with the args passed in as is.